src/lib.ml

```diff
@@ -388,34 +390,36 @@ module L = struct
         List.fold_left ~f:Path.Set.add ~init:acc
           [public_cmi_dir ; native_dir])
     in
-    Path.Set.remove dirs stdlib_dir
+    match ts with
```

Comment:
This is an optimisation right? I'm not sure it's worth it: removing something from an empty set should be really fast

Reply:
Nah, the stdlib_dir itself comes from a library so we don't what to remove unless we have at least 1 lib in the closure.

Reply:
Oh, I see